services/src/test/scala/cromwell/services/womtool/impl/WomtoolServiceInCromwellActorSpec.scala

```diff
@@ -195,15 +197,15 @@ class WomtoolServiceInCromwellActorSpec extends ServicesSpec("Womtool") {
       )
     }
 
-    "return invalid for a valid no-inputs workflow with extraneous inputs" in {
+    "return valid = false, validWorkflow = true for a valid no-inputs workflow with extraneous inputs" in {
 
       val wsfc = wsfcConjurer(workflowSource = Option(TestData.wdlValidNoInputs), inputsJson = TestData.bogusInputs)
 
       check(DescribeRequest(wsfc), DescribeSuccess(
-        description = WorkflowDescription(valid = false, errors = List("WARNING: Unexpected input provided: foo.bar"))))
+        description = WorkflowDescription(valid = false, errors = List("WARNING: Unexpected input provided: foo.bar"), validWorkflow = true, name = "wf_hello", inputs = List.empty, submittedDescriptorType = Map("descriptorType" -> "WDL", "descriptorTypeVersion" -> "1.0"))))
```

Comment:
I thought these were being put into a separate `inputErrors` field (per the PR title?)

Or are we now saying "if you see `validWorkflow` but still have errors, all errors indicate input errors"?

Reply:
Good catch, the PR title is out of date.